### PR TITLE
add CI build for just DBAL without ORM

### PR DESCRIPTION
**Pull request by dmaicher**

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -46,6 +46,12 @@ jobs:
             dependencies: "highest"
             php-version: "8.2"
 
+          # DBAL only without ORM
+          - php-version: "8.2"
+            dependencies: "highest"
+            stability: "stable"
+            remove-orm: true
```
Comment (stof) [inline .github/workflows/continuous-integration.yml:55]: See my comment on #1707 about the matrix.

**Review by stof — APPROVED** [review]

**Review by ostrolucky — APPROVED** [review]
Should he been part of 2.10.x though

**Comment by dmaicher** [comment]
> Should he been part of 2.10.x though

I can also rebase it on 2.10.x :+1:

**Comment by dmaicher** [comment]
Hm interesting test fails

```
 1) Doctrine\Bundle\DoctrineBundle\Tests\ProfilerTest::testRender
Twig\Error\RuntimeError: Variable "profile_type" does not exist.

/home/runner/work/DoctrineBundle/DoctrineBundle/vendor/symfony/web-profiler-bundle/Resources/views/Profiler/layout.html.twig:5
/home/runner/work/DoctrineBundle/DoctrineBundle/vendor/twig/twig/src/Template.php:171
/home/runner/work/DoctrineBundle/DoctrineBundle/vendor/twig/twig/src/Template.php:394
/home/runner/work/DoctrineBundle/DoctrineBundle/vendor/twig/twig/src/Template.php:367
/home/runner/work/DoctrineBundle/DoctrineBundle/vendor/twig/twig/src/Template.php:394
/home/runner/work/DoctrineBundle/DoctrineBundle/vendor/twig/twig/src/Template.php:367
/home/runner/work/DoctrineBundle/DoctrineBundle/vendor/twig/twig/src/Template.php:394
/home/runner/work/DoctrineBundle/DoctrineBundle/vendor/twig/twig/src/Template.php:367
/home/runner/work/DoctrineBundle/DoctrineBundle/vendor/twig/twig/src/Template.php:379
/home/runner/work/DoctrineBundle/DoctrineBundle/vendor/twig/twig/src/TemplateWrapper.php:40
/home/runner/work/DoctrineBundle/DoctrineBundle/vendor/twig/twig/src/Environment.php:280
/home/runner/work/DoctrineBundle/DoctrineBundle/Tests/ProfilerTest.php:97
``` 

maybe related to the CLI profiler changes on 6.4? :thinking:

**Pull request merged**